src/NumberGuessingGame.js

```diff
@@ -55,6 +55,4 @@ function getRandomNumber() {
 
 const MAX_ATTEMPTS = 5;
```

Comment:
`MAX_ATTEMPTS` and `getRandomNumber` should be placed above the component. You can even put it inside since nothing else is using it besides the component.